sass/components/_announcement-bar.scss

```diff
@@ -90,23 +88,95 @@
         font-size: 16px;
     }
 
-    button {
-        &:hover {
-            background-color: rgba(255, 255, 255, 0.08);
-        }
-
+    .announcement-bar__btn {
         height: 24px;
         box-sizing: border-box;
-        padding: 4px 8px;
+        padding: 6px 10px;
         border: 1px solid #fff;
-        margin-left: 8px;
-        background-color: inherit !important;
+        background-color: inherit;
         border-radius: 4px;
         font-family: Open Sans;
-        font-size: 12px;
+        font-size: 11px;
         font-style: normal;
         font-weight: 600;
-        line-height: 16px;
+        line-height: 10px;
+
+        &.announcement-bar__btn--secondary {
+            &:hover {
+                background-color: rgba(255, 255, 255, 0.08);
+            }
+
+            &:active {
+                background-color: rgba(255, 255, 255, 0.16);
+            }
+
+            &:focus {
+                border-color: transparent;
+                outline: 2px solid var(--sidebar-text-active-border);
+            }
+        }
+
+        &.announcement-bar__btn--tertiary {
+            border: none;
+            color: var(--sidebar-text-rgb);
```

Comment:
This might be incorrect since it'll set the colour to something like `color: 255, 255, 255;`. It should probably be using the non-rgb version of that variable

Reply:
Good catch, you are right. Fixed.